conf/capsule.yaml.template

```
@@ -13,6 +10,8 @@ CAPSULE:
     # The base os rhel version where the capsule installed
     # RHEL_VERSION:
   # The Ansible Tower workflow used to deploy a capsule
-  DEPLOY_WORKFLOW: deploy-sat-capsule
+  DEPLOY_WORKFLOW: deploy-capsule
   # Dictionary of arguments which should be passed along to the deploy workflow
-  # DEPLOY_ARGUMENTS:
+  DEPLOY_ARGUMENTS:
+    # Flavor for big capsules deployment
+    LARGE_FLAVOR: satqe-ssd.disk.xxl
```

Your change would add `large_flavor: satqe-ssd.disk.xxl` extra variable to each and every capsule deployment.
Consider defining a list of available flavors somewhere in settings, not necessarily in `capsule.yaml` or in constants, as it could be used by other host types.
```suggestion
  DEPLOY_FLAVORS:
    LARGE: satqe-ssd.disk.xxl  # Flavor for big capsules deployment
  DEPLOY_ARGUMENTS:
```
This is the only blocking comment in this review.

---

Great catch! I'll update update it in the comming week.
Thanks @ogajduse !